### ISSUE TITLE
4.0 beta 1, New lazyload image feature, conflict with performance plugins (Perfmatters)

### ISSUE THREAD

**Issue by madjedo** [soflyy/oxygen-bugs-and-features#2748]
**Describe the bug**
The introduction of lazyloading is causing conflicts with performance plugins like Perfmatters. 2 different lazy load classes/attributes is being set on each image. Native wordpress/browser tags and perfmatters lazyload classes. Perfmatters disables native wordpress lazyloading. There should be an option in Oxygen back-end to enable/disable the lazy load feature (preferable under 'bloat eliminator' tab, recommended name change to: 'Performance control').

But good job of finally introducing native lazyload support. Thank you. But not needed in all cases im afraid.

**Comment by Spellhammer** [comment]
Are you saying that the loading="eager" attribute is conflicting with the Perfmatters classes?

No images should ever have loading="lazy" unless you've ticked the lazy loading box in Oxygen.

**Comment by madjedo** [comment]
> Are you saying that the loading="eager" attribute is conflicting with the Perfmatters classes?
> 
> No images should ever have loading="lazy" unless you've ticked the lazy loading box in Oxygen.

Actually lazy loading still works with Perfmatters despite having "eager" attribute set.

I had another issue, the issue was Perfmatters have an option to set image attributes for width & height. I use this feature and appreciate it a lot.

Well, this feature no longer works. And I tried to reproduce it on a sandbox, but I couldn't! I thought  that was odd. But with additional testing I was finally able to reproduce the bug:

1. Install Oxygen 3.9
2. Create a page, upload image with image element (media library or image url option doesnt matter)
3. Install and activate perfmatters, 
4. Activate setting "set missing image width and height dimensions" under tab "lazy load".
5. Visit front-end, enter dev-tools, inspect image element, width and height should be added correctly.
6. Now update Oxygen to 4.0 beta 1, visit front-end again, width and height attributes now gone.

Basically, one would have to re-upload every single image for the Perfmatters plugin to work correctly for this.

Is it related to this? Maybe? Let me know if you want a new post about this and I can go ahead and do another one for this specific issue or if this is a Perfmatters "thing".

**Comment by KittenCodes** [comment]
@madjedo I've requested a copy of Perfmatters so that we can try to reproduce this.
